src/modules/Search/Search.d.ts

```diff
@@ -61,11 +62,10 @@ export interface StrictSearchProps {
   /**
    * Renders the SearchCategory layout.
    *
-   * @param {object} categoryContent - The Renderable SearchCategory contents.
-   * @param {object} resultsContent - The Renderable SearchResult contents.
+   * @param {object} props - The SearchCategoryLayout props object.
    * @returns {*} - Renderable SearchCategory layout.
    */
-  categoryLayoutRenderer?: (props: SearchCategoryProps) => React.ReactElement<any>
+  categoryLayoutRenderer?: (props: SearchCategoryLayoutProps) => React.ReactElement<any>
```

Comment:
```suggestion
  categoryLayoutRenderer?: (props: Pick<SearchCategoryLayoutProps, 'categoryContent' | 'resultsContent'>) => React.ReactElement<any>
```

Same as in other comment :)